### ISSUE TITLE
Question on generated classes and kapt

### ISSUE THREAD

**Issue by ftomassetti** [sgdan/kiss#1]
hi, my understand was that to use Truffle one had to configure the annotation processor tool so that it generates some classes. Is this true? If this is the case, how you handle this in this project?

**Comment by sgdan** [comment]
Yes, that's in the gradle build file: https://github.com/sgdan/kiss/blob/master/build.gradle.kts

```
    kotlin("kapt") version "1.3.60"
```
```
    kapt("org.graalvm.truffle:truffle-dsl-processor:19.2.1")
```

Btw it's a while since I created this example, you might be interested in a more recent example where I tried to compile some Elm code to a native executable via Truffle: https://github.com/sgdan/truffle-elm
That's closer to a real use case than this project, and uses version 19.3.0 of the Truffle API. The code with the annotations is https://github.com/sgdan/truffle-elm/blob/master/src/main/kotlin/org/sgdan/grelm/TruffleNodes.kt
